packages/@react-aria/dnd/src/DragManager.ts

```diff
@@ -517,6 +517,7 @@ class DragSession {
   }
 
   cancel() {
+    setDropCollectionRef(undefined);
```

Comment:
If I don't clear it here, dragEnd could potentially return `isInternalDrop = true` since the dropCollectionRef is set on dropEnter and isn't cleared until calling onDragEnd. The `isInternalDrop` being set to `false` here probably isn't a super important thing, just felt kinda weird, happy to remove if you think the above is unnecessary

Reply:
hmm couldn't onDragEnd check if the dropOperation is 'cancel'? Otherwise, this could also affect mouse dragging right?

Reply:
Ah yes, I guess I could, will update in the followup pr. As for affecting mouse dragging, this is DragManager specific so it shouldn't affect mouse dragging right?